README.md

```diff
@@ -2,152 +2,46 @@
 
-All tools should build (along with any external packages) and run very easily, provided you are able to use CMake. CMake should always work similarly, but the easiest methods may vary by operating system.
+## Build and Run
 
-The basic steps (more specific instructions for Windows are below):
+The easiest way to build and run the benchmarks is to [use Docker](docs/Docker.md). If that doesn't work for you, please, refer to our [build and run guide](docs/BuildAndTest.md).
 
-1) Clone the repository (make sure submodules are cloned properly)
-2) Optional: set HUNTER_ROOT environment variable to choose where HunterGate pacakges are stored.  If not set, will default to etc/HunterGate-Root.
-3) Run cmake
-	- mkdir my-build  # Directories with name *-build/ are ignored in .gitignore
-	- cd my-build
-	- cmake ..
-	- If you get an error of the form "The input line is too long.  The syntax of the command is incorrect.", your $ENV:PATH may be too long (!)
+## Plot Results
 
-3) Build
+Use `ADBench/plot_graphs.py` script to plot graphs of the resulting timings.
 
-### Windows
-
-The easiest way to build on windows is using [CMake tools for Visual Studio](https://blogs.msdn.microsoft.com/vcblog/2016/10/05/cmake-support-in-visual-studio/), which can be installed as a component in the [Visual Studio Installer](https://docs.microsoft.com/en-us/visualstudio/install/install-visual-studio).
-
-1) Open Visual Studio
-2) [Clone](http://www.malgreve.net/2014/06/17/cloning-getting-code-from-git-repository-to-visual-studio/) this repository
-3) Open the cloned folder in Visual Studio. CMake should run automatically, but it may need to be started manually.
-	- **NOTE:** CMake (specifically HunterGate) often seems to crash the first time it is run, so try re-running several times.
-	- **NOTE:** HunterGate requires that the path to the folder does not contain spaces. This can be disabled but is not currently.
-4) When CMake has finished, run CMake>Build All
-	- **NOTE:** this is sometimes not available as an option. Try restarting Visual Studio or waiting a while.
-
-All tools should now be built. See [Usage](#usage) below.
-
-#### Windows from the command line
-
-Instead of using Visual Studio you can execute the following command
-
-```
-cmake -G "Ninja" '-DCMAKE_TOOLCHAIN_FILE=<path-to-top-level>\toolchain.cmake' '-DCMAKE_BUILD_TYPE="RelWithDebInfo"' "<path-to-top-level>"
-ninja
+```bash
+python ADBench/plot_graphs.py [--save] [--plotly] [--show]
 ```
+- `--save`: save graphs as .png files to `tmp/graphs/static/`
+- `--plotly`: save graphs as [Plot.ly](https://plot.ly/) .html files to `tmp/graphs/plotly/`
+- `--show`: display graphs in new windows.
 
-You have to somehow ensure that `cmake`, `cl` and `ninja` are on your
-path.
+If neither `--save` nor `--plotly` are included, `--show` will be `True` by default - otherwise, it must be manually enabled.
```

Comment:
Maybe these lines should be removed at all as far as a reference to the plot creating documentation file is here. In my opinion, duplicating of documentation should be minimized

Reply:
OK, let's have a single example of call in README and the complete explanation in the corresponding separate doc.

Reply:
Ok